app/assets/stylesheets/application.css

```diff
@@ -0,0 +1,147 @@
+body {
+    font-family: Arial, Helvetica, sans-serif;
+    font-size: 1em;
```

Comment:
@tomerdesserww  that's a common default.
<img width="648" alt="screen shot 2018-10-25 at 11 01 05" src="https://user-images.githubusercontent.com/1285305/47484814-524e3400-d845-11e8-8ac8-3bb3a0351afb.png">

But the fact the everywhere else Eti used pixels so it's really doesn't even matter, because most of the advantages of the `em` unit is not used.